### PR TITLE
Test on Symfony 3.3 and integrate phpunit-bridge

### PR DESCRIPTION
**Pull request by Koc**
two questions before merge
1. do we need leave 3.2 also or even 3.1 because of composer reqs?
2. do we need add test env without full-stack and only for framework-bundle (or `--prefer-lowest`)?

### CODE REVIEW

File: Tests/Vich/UploaderBundle/Tests/Form/Type/VichFileTypeTest.php [hunk 1]
```diff
@@ -21,8 +21,6 @@ class VichFileTypeTest extends TestCase
 
     /**
      * @dataProvider configureOptionsBCDataProvider
-     * @group legacy
-     * @expectedDeprecation The "download_link" option is deprecated since version 1.6 and will be removed in 2.0. You should use "download_uri" instead.
```
Comment (Koc) [inline Tests/Vich/UploaderBundle/Tests/Form/Type/VichFileTypeTest.php]: removed just for testing that build fails without this annotations

**Comment by Koc** [comment]
@garak ready, please review

File: Tests/PHPUnitTestCase.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (garak) [inline Tests/PHPUnitTestCase.php]: What's the purpose of this file?

File: composer.json [hunk 3]
```diff
@@ -42,7 +42,9 @@
         "symfony/browser-kit": "^2.8|^3.1",
         "symfony/css-selector": "^2.8|^3.1",
         "symfony/dom-crawler": "^2.8|^3.1",
+        "symfony/validator": "^2.8|^3.1",
```
Comment (garak) [inline composer.json]: unsorted

File: Tests/PHPUnitTestCase.php (hunk 2 above)
Comment (Koc) [inline Tests/PHPUnitTestCase.php]: It required for test pass on `2.8.0` when composer run with `--prefer-lowest`. Symfony added support of the namespaced PHPUnit from 2.8.18

Comment (garak) [inline Tests/PHPUnitTestCase.php]: So it's better to raise requirements for symfony to `^2.8.18`. There's no value in supporting older versions

Comment (Koc) [inline Tests/PHPUnitTestCase.php]: If we can do it - of course, yes

Comment (garak) [inline Tests/PHPUnitTestCase.php]: go for it

**Comment by garak** [comment]
There's no need to drop Symfony 3.1 and 3.2 for now, even if we require the phpunit bridge for 3.3 (since it's compatible with previous Symfony versions).
Anyway, we need to investigate on the failure for the lowest requirements and see if possibly increase some dependencies' minimum versions

**Comment by Koc** [comment]
@garak tests are green finally now

**Pull request merged**